src/main/resources/assets/admin/common/js/form/set/FormSetOccurrenceView.ts

```diff
@@ -76,9 +76,9 @@ export class FormSetOccurrenceView
             return false;
         });
 
-        this.label = new FormOccurrenceDraggableLabel(this.getFormSet().getLabel(), this.getFormSet().getOccurrences());
+        this.label = new FormOccurrenceDraggableLabel(this.getFormSet().getLabel(), this.getFormSet().getOccurrences(), this.getFormSet().getLabel());
```

Comment:
Codacy found an issue: [Exceeds maximum line length of 140](https://app.codacy.com/manual/enonic/lib-admin-ui/pullRequest?prid=6371096)